pkg/cli/tk/listbox.go

```diff
@@ -284,6 +294,12 @@ func (w *listBox) Handle(event term.Event) bool {
 	case term.K(ui.Enter):
 		w.Accept()
 		return true
+	case term.K(ui.Up, ui.Alt):
+		w.OnUnderFlow(w)
+		return true
+	case term.K(ui.Down, ui.Alt):
+		w.OnOverFlow(w)
+		return true
```

Comment:
It would be nicer if this is part of the keybinding rather than hardcoded in the event handler.